pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -7513,11 +7513,12 @@ spec:
   params:
     - name: platform
     - name: browser
+    - name: version
   steps:
     - name: echo
       image: alpine
       script: |
-        echo "$(params.platform) and $(params.browser)"
+        echo "$(params.platform) and $(params.browser)" and $(params.version)" 
```

Comment:
I don't think it matters in practice, but there is an extra double-quote here.

Reply:
good catch, fixed it - thanks!